tests/read_mac/test_read_mac_metadata.py

```diff
@@ -0,0 +1,146 @@
+import pytest
+import logging
+
+from tests.common.utilities import wait
+from tests.common.utilities import wait_until
+from tests.common.plugins.sanity_check import checks
+from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer
+from tests.common.helpers.assertions import pytest_assert
+from datetime import datetime
+
+
+logger = logging.getLogger(__name__)
+
+BINARY_FILE_ON_LOCALHOST = "/tmp/update_image.bin"
+BINARY_FILE_ON_DUTHOST = "/tmp/mellanox.bin"
```

Comment:
It looks like this script is not Mellanox specific, suggest to use more general image name here.

Reply:
Fixed